lms/templates/certificates/shipping_information.html

```diff
@@ -0,0 +1,81 @@
+<%page expression_filter="h"/>
```

Comment:
It probably makes sense to have someone from UX take a quick pass at this before launching. We could probably do some very simple stuff that would make this look 10x better!

Reply:
+1 - but it is a test, so grain of salt rule applies.

Reply:
also this shouldn't block starting the test since no one will be earning a certificate and seeing this page in the first few weeks